errors/errors.go

```diff
@@ -144,3 +145,12 @@ func FromError(err error) *Error {
 
 	return Parse(err.Error())
 }
+
+// As finds the first error in err's chain that matches *Error
+func As(err error) (bool, *Error) {
```

Comment:
Should return (*Error, bool) ?

Reply:
Yeah, it's better to be comma, ok style, already updated.